helpdesk_mgmt/views/helpdesk_ticket_tag_views.xml

```diff
@@ -49,10 +49,10 @@
         <field name="name">helpdesk.ticket.tag.tree</field>
         <field name="model">helpdesk.ticket.tag</field>
         <field name="arch" type="xml">
-            <tree decoration-muted="not active">
+            <tree decoration-muted="not active" editable="bottom">
                 <field name="name" />
                 <field name="color" widget="color_picker" />
-                <field name="active" widget="boolean_toggle" />
+                <field name="active" invisible="1" />
```

Comment:
Why changing this?

Reply:
I prefer to leave the existing behavior in other tree views of active by selecting the records and using the corresponding action. 

Would you prefer we keep this with `boolean_toggle`?

Reply:
OK, as the inactive are not shown by default.